kuzzleio/kuzzle-admin-console
# UI/UX tweaks to make collection-creation workflow more human

## What does this PR do ?
This PR introduces some UI tweaks to make the collection-creation wizard more human. This may just be the beginning of a process, if you have ideas and feedback, they are welcome.

### How should this be manually tested?

* click on the create collection button
* specify a name and no mapping
* you'll see a message explaining what happens

Or,

* click on the create collection button
* specify a name and a mapping
* you'll see a new, revisited wizard step, with explaining text and alternative UI.

### Other changes

* Styles contained in `assets/collection.scss` and `assets/document.scss` have been dispatched to components and the file has been deleted. Code is still NOT BEM-compliant.
* The Mapping card in the CreateOrUpdateDocument component contains no-longer a read-only json-editor.

### Screenshots

#### Before

![selection_125](https://user-images.githubusercontent.com/5023426/44790918-b97e9d80-aba0-11e8-8325-9269154961ee.png)

![selection_128](https://user-images.githubusercontent.com/5023426/44791108-314cc800-aba1-11e8-9966-e3c95d24452e.png)


#### After

![selection_126](https://user-images.githubusercontent.com/5023426/44790981-e632b500-aba0-11e8-9a4d-8e433d9fab30.png)

![selection_127](https://user-images.githubusercontent.com/5023426/44791050-0f534580-aba1-11e8-9717-6bc3f520815e.png)
